server/db/migrations/19_create_host_node_stats_created_at_index.rb

```diff
@@ -0,0 +1,5 @@
+class CreateHostNodeStatsCreatedAtIndex < Mongodb::Migration
+  def self.up
+    HostNodeStat.create_indexes
```

Comment:
Migration number 19 is already used in #1874, so prepare to change (depending on which one got merged first)